### PR TITLE
add avatar component, icon element

### PR DESCRIPTION
**Pull request by ritik-patel05**
https://user-images.githubusercontent.com/67374176/151677223-4f38eadd-c708-4710-bdcd-1bfceaee3e08.mp4

### CODE REVIEW

File: css/elements/icon.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.icon {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--size-11);
+    height: var(--size-11);
+}
```
Comment (Hamdrive) [inline css/elements/icon.css]: Add newline to EOF

**Review by Hamdrive — COMMENTED** [review]
LGTM

File: css/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    font-size: var(--size-13);
+}
+
+/* ---- Image avatars ---- */
+.avatar.is-img {
+    object-fit: cover;
+    color: transparent;
+
+    /* Some large value, so that no text is shown */
+    text-indent: 10000px;
```
Comment (saadahmadhere) [inline css/components/avatar.css]: Good to see clean and descriptive css. Text indent is used to define the indentation of first block of text. Margins here would me more appropriate.

**Review by saadahmadhere — COMMENTED** [review]
Nice work.

File: css/components/avatar.css (hunk 2 above)
Comment (ritik-patel05) [reply css/components/avatar.css]: removed it for now

**Pull request merged**